app/scripts/lib/constants.js

```diff
@@ -24,6 +24,9 @@ define([], function () {
 
     OAUTH_CODE_LENGTH: 64,
 
+    DERIVED_KEY_LENGTH: 32,
+    DERIVED_KEY_INFO_PREFIX: 'identity.mozilla.com/picl/v1/oauth/',
```

Comment:
I could see the derived relier key module be extracted out at some point. Moving the constants to that module/file would make sense and also help clarify the context issue as well. Any other particular reason for putting them here?